### ISSUE TITLE
Drupal 7 Upgrade Event in conjunction with Backdrop LIVE

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdrop-community#31]
Background - we've already been in discussions with the Drupal Association about planning and sponsoring a online event targeted at Drupal 7 site owners to help them educate themselves about the EOL schedule and their upgrade options, including but not limited to Modern Drupal and Backdrop CMS. 

PROPOSAL: It has been suggested that we connect this loosely with the next Backdrop LIVE event. Current thinking is that the first block of our next Backdrop LIVE, be repurposed into a different Drupal 7 upgrade event coordinated with Backdrop LIVE. Example: First block on Friday morning is the Drupal 7 upgrade event followed by Backdrop LIVE starting in Block B. 

Does anyone object to this kind of coordination?
Does anyone have additional ideas on how to make this work?

Does anyone have any questions?

Discussion in Drupal Soft Landing queue
https://www.drupal.org/project/d7_eol_soft_landing/issues/3348602

**Comment by cellear** [comment]
I think it's a great idea.

**Comment by yorkshire-pudding** [comment]
Good idea. If people attending that are interested, there will be lots of topics to help them go further.

**Comment by stpaultim** [comment]
No one has objected to this. I got some feedback that maybe we should schedule this a week before Backdrop LIVE, so that folks could make plans to attend Backdrop LIVE the following week. 

It's probably too late for this. I think the next step is a proposal for what this upgrade event is going to look like. We might need more than 4 weeks to arrange all the necessary partners. 

@irinaz - What do you think?

**Comment by irinaz** [comment]
@stpaultim , let's discuss format  and timing - TwinCities DrupaCamp is week before BackdropLive, right?
